abrute

```diff
@@ -0,0 +1,74 @@
+.\" Manpage for abrute
+.\" Contact alcamech@gmail.com to correct errors or typos.
+.TH man 1 "12 October 2017" "1.0" "abrute man page"
+.SH NAME
+abrute \- Multi-threaded AES Brute Force File Decryption
+.SH SYNOPSIS
+abrute <RANGE> <CHARACTERS> [OPTIONS] -- <TARGET>
+.SH DESCRIPTION
+Multi-threaded AES Brute Force File Decryption
+.SH OPTIONS
+<RANGE>         Single digit or a range 4:6 for password length.
+.PP
+<CHARACTERS>    Characters to use in password attempt. Don't use quotes
+                unless they may be in the password. Backslash may escape
+                characters such as space.
+.PP
+-a, --adjacent  Set a limit for allowed adjacent characters. Zero will
+                not allow any characters of the same kind to neighbor
+                in the attempts.
+.PP
+-s, --start     Starting character sequence to begin with.
+.PP
+-z, --zip       Use `unzip` decryption instead of `aescrypt`.
+.PP
+<TARGET>        Target file to decrypt.  The target must be preceeded
+                by a double dash: -- target.aes
+.PP
+-h, --help      Prints help information.
+.PP
+-v, --version   Prints version information.
+.SH EXAMPLES
+Decrypts a file with a range of 4, characters of 1234, and a target path to the
+example.file.aes
+
+abrute 4 1234 -- test/data/example.file.aes 1>/dev/null
```

Comment:
You can remove ` 1>/dev/null` off of each example

Reply:
Also you can remove `test/data/`

Reply:
done